src/js/components/CourseComponent.js

```diff
@@ -258,6 +258,7 @@ module.exports = React.createBackboneClass({
                 onBlur={this.blurGrade}
                 data-student-id={student.id}
                 data-grade-name={grade.name}
+                value={grade.url}
```

Comment:
the grade for the assignment is supposed to go here. The url is attached with a button if available